src/components/global/BaseTag.vue

```diff
@@ -0,0 +1,81 @@
+<script>
+import { defineComponent } from 'vue';
+
+import TagEllipse from '@/assets/icons/tag-ellipse.svg';
+
+export default defineComponent({
+    name: 'BaseTag',
+    components: {
+        TagEllipse,
+    },
+    props: {
+        labels: {
+            type: [String],
```

Comment:
this is equivalent to just
```js
type: String
```
If you want it to be an array you need to define it as
```js
type: Array
```

if you want to define it as array of strings you need to use TypeScript for that (and I'm sure we will get to how to do it in Vue one day).

Reply:
Ok, this may have been a bit of a guess here..
Thanks for the explanation though!

Reply:
Your syntax is also a valid option but array of types is used when you want to allow multiple types
https://vuejs.org/guide/components/props.html#prop-validation